.github/workflows/release.yml

```diff
@@ -222,6 +222,27 @@ jobs:
     needs: [ build, version-tag ]
     if: needs.version-tag.outputs.append_notes == 'true'
     steps:
```

Comment:
- Consider adding error handling for the case where the release or its body is invalid to provide better feedback.